pandas/core/groupby/ops.py

```diff
@@ -124,6 +129,8 @@ def __init__(self, kind: str, how: str):
         },
     }
 
+    _MASKED_CYTHON_FUNCTIONS = {"cummin", "cummax"}
```

Comment:
why is a separate variable necessary here? that is just confusing

Reply:
I think it makes sense to store this information in the `WrappedCythonOp` class, instead of putting this in `_cython_operation` (which would be your alternative?)

Reply:
yes this would be fine, its not clear that this happening in this PR